packages/client/src/forms/__tests__/getFormSchema.test.ts

```diff
@@ -32,6 +32,7 @@ describe('schemas client', () => {
       ).resolves.toEqual(successResponse);
 
       expect(spy).toHaveBeenCalledWith(
+        // @ts-expect-error
```

Comment:
This `// @ts-expect-error` should not be added as the real problem is with the type `FormSchemaQuery` that should be defined like this:

```js
export type FormSchemaQuery = {
  includeJsonSchema?: 'true' | 'false';
  resolveJsonSchemaPresets?: 'true' | 'false';
};
```

This is because of the typings used for the `proper-url-join` which do not accept boolean values for the `query` property. Can you change that type and remove any unnecessary `// @ts-expect-error` directives that might result of that change?

Reply:
As discussed offline, we will keep as is.